yarn.lock

```diff
@@ -1438,8 +1640,9 @@ cheerio@^1.0.0-rc.2:
     parse5 "^3.0.1"
 
 chokidar@^2.0.0, chokidar@^2.0.2:
-  version "2.0.3"
-  resolved "https://registry.yarnpkg.com/chokidar/-/chokidar-2.0.3.tgz#dcbd4f6cbb2a55b4799ba8a840ac527e5f4b1176"
+  version "2.0.4"
+  resolved "https://registry.yarnpkg.com/chokidar/-/chokidar-2.0.4.tgz#356ff4e2b0e8e43e322d18a372460bbcf3accd26"
```

Comment:
This is the main change, 2.0.4 chokidar to fix it: 
<img width="1069" alt="screen shot 2019-01-08 at 12 18 45 pm" src="https://user-images.githubusercontent.com/10549495/50824603-b4e0a780-133f-11e9-98bb-265b17835ab2.png">